cli/project/dockerfiles.go

```diff
@@ -181,11 +183,40 @@ func getInstallTarantoolLayers(ctx *context.Ctx) (string, error) {
 	return installTarantoolLayers, nil
 }
 
+func getOpenSourceInstallerLink(version string) string {
+	parts := strings.SplitN(version, ".", 3)
+	major, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return ""
+	}
+
+	minor, err := strconv.Atoi(parts[1])
+	if err != nil {
+		return ""
+	}
+
+	if major >= 2 && minor > 8 {
+		preReleasePostfix := []string{"-alpha", "-beta", "-rc"}
+		for _, postfix := range preReleasePostfix {
+			if strings.Contains(version, postfix) {
+				return preReleaseOpenSourceInstaller
+			}
+		}
+
+		return releaseOpenSourceInstaller
+	}
+
+	return defaultOpenSourceInstaller
+}
+
 const (
 	DefaultBaseBuildDockerfile   = "Dockerfile.build.cartridge"
 	DefaultBaseRuntimeDockerfile = "Dockerfile.cartridge"
 
-	containerSDKPath = "/usr/share/tarantool/sdk"
+	containerSDKPath              = "/usr/share/tarantool/sdk"
+	defaultOpenSourceInstaller    = "https://tarantool.io/installer.sh"
+	preReleaseOpenSourceInstaller = "https://tarantool.io/pre-release/2/installer.sh"
+	releaseOpenSourceInstaller    = "https://tarantool.io/release/2/installer.sh"
```

Comment:
We can use the first URL for all cases:

1. `tarantool <= 2.8`: `VER=X.Y installer.sh --type release|live`.
2. `tarantool > 2.8`: `VER=X installer.sh --type release|pre-release`.

I think it'll simplify the logic in the code.

Reply:
Fixed